packages/client/src/utils/gmodel-util.ts

```diff
@@ -253,7 +254,9 @@ export type MoveableElement = GModelElement & Locateable;
 
 export interface Resizable extends BoundsAware, Selectable {}
 
-export interface ResizableModelElement extends GParentElement, Resizable {}
+export interface ResizableModelElement extends GParentElement, Resizable {
+    resizeLocations?: ResizeHandleLocation[];
```

Comment:
We should mirror this interface to  the server side and the `GShapedElement` should implement it by default.
Otherwise its not possible to define the resize location in the model on the server side.

Reply:
That is a good point, as soon as this PR is merged, I'll open one for our two server repositories.